acquia_lift.admin.wizard.inc

```diff
@@ -840,6 +933,9 @@ function acquia_lift_personalize_campaign_wizard_submit(&$form, &$form_state) {
     $submit_function($form, $form_state, $agent_data, $agent_instance);
   }
 
+  if (isset($form_state['values']['variations']['variation_set_handling'])) {
+    $agent_data->data['variation_set_handling'] = $form_state['values']['variations']['variation_set_handling'];
+  }
```

Comment:
I think my only reason was that this is where changes to the agent are being saved. Otherwise we need to call personalize_agent_save() again in the variations submit handler.